acceptance/tests/resource/service/service_enable_linux.rb

```diff
@@ -13,9 +13,6 @@
                 'ubuntu' => 'apache2',
 }
```

Comment:
I'd lean towards confining it to the packages listed here, rather than excepting everything else we know about. Less work to maintain.